scripts/linkbot/monitor-raspbian-os.js

```diff
@@ -35,13 +35,13 @@ function getNewRaspbian(target, url, separator) {
         var document = dom.window.document;
         var refs = document.getElementsByTagName("a");
         var test = Array.from(refs)
-            .filter((ref) => ref.textContent.match("raspios-.*-lite.*.zip$"))
+            .filter((ref) => ref.textContent.match("raspios-.*-lite.*.(zip|xz)$"))
             .reduce((acc, element) => {
-                acc.push(element.textContent.match("raspios-.*-lite.*.zip$").input);
+                acc.push(element.textContent.match("raspios-.*-lite.*.(zip|xz)$").input);
                 return acc;
             }, [])[0];
         console.log(`Test var: ${test}`);
-        const imageName = test.substring(0, test.length - 4);
+        const imageName = test.replace(/\.img\..*/, "");
```

Comment:
```suggestion
        const imageName = test..substring(0, test.lastIndexOf('.img'));
```
since the regex might fail if the next time the image is provided as a plain `.img`

Reply:
Uhh, smrt!